ports/stm32/boards/MIKROE_QUAIL/pins-zerynth.csv

```diff
@@ -0,0 +1,211 @@
+# Pin mapping according to Zerynth documentation
```

Comment:
I'm not sure this file should be included, there's really no reason to match Zerynth, code from there will not run in MicroPython.

Pins should be named based on the labels on the physical board and/or the board documentation.

Reply:
OK, I'll do that.